src/Sylius/Component/Core/Dashboard/SalesDataProvider.php

```diff
@@ -13,51 +13,67 @@
 
 namespace Sylius\Component\Core\Dashboard;
 
-use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
 use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\OrderPaymentStates;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
 
 /**
  * @experimental
  */
 final class SalesDataProvider implements SalesDataProviderInterface
 {
-    /** @var EntityManagerInterface */
-    private $entityManager;
+    /** @var OrderRepositoryInterface|EntityRepository */
```

Comment:
Do we need both definitions because of `createQueryBuilder`? Maybe we should declare make it available through the main interface? But that would couple us with Doctrine ORM probably. /cc @pamil @Zales0123

Reply:
If it's allowed by our BC policy then I suppose 👍

Reply:
Ref. https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html?%20policy#repositories-repository-interfaces